tests/pyccel/scripts/kernel/hello_kernel.py

```diff
@@ -0,0 +1,4 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+from func import f
+f()
```

Comment:
Is there a use to having this here rather than in a main in the file `func.py`?

Reply:
Yes

Reply:
If there is no main then there may not be an executable so the test probably won't work in this case. Separately I don't see any need for an extra file and don't want to increase the number of files just because we can

Reply:
can u check now 4307d67 i remove all  unnecessary lines here 2ce3bf2

Reply:
can u please check now 70e7cdf

Reply:
This looks right :+1: